modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css

```
@@ -7,19 +7,38 @@
 }
 
 .hero-container__image {
-	background-image: url('[resources:bg_hero.svg]');
-	background-repeat: no-repeat;
-	background-size: 100% 95%;
-	border-radius: 0.75rem;
+	height: 217px;
+	position: absolute;
+	width: 100%;
```

these two can be convertible on bootstrap

```suggestion
	position-absolute and w-100
```